code/camera/camera.cpp

```diff
@@ -145,7 +145,7 @@ void camera::set_object_target(object *objp, int n_object_target_submodel)
  * Custom function receives the already-modified current position value.
  * It should be replaced or added to as the custom function modifier sees fit.
  */
-void camera::set_custom_position_function(void (*n_func_custom_position)(camera *cam, vec3d *camera_pos))
```

Comment:
For function pointer types it would be better to leave the parameter names in place to document what the parameters mean.